components/eventbrite/actions/locales.js

```diff
@@ -0,0 +1,30 @@
+module.exports = [
+  "de_AT",
+  "de_CH",
+  "de_DE",
+  "en_AU",
+  "en_CA",
+  "en_DK",
+  "en_FI",
+  "en_GB",
+  "en_KH",
+  "en_IE",
+  "en_IN",
+  "en_NZ",
+  "en_SE",
+  "en_US",
+  "es_AR",
+  "es_CL",
+  "es_CO",
+  "es_ES",
+  "fr_BE",
+  "fr_CA",
+  "fr_CH",
+  "fr_FR",
+  "hi_IN",
+  "it_IT",
+  "nl_BE",
+  "nl_NL",
+  "pt_BR",
+  "pt_PT",
+];
```

Comment:
Is there an endpoint in Eventbrite's API from which we could retrieve these codes? Or are they documented somewhere?

An easier alternative would be to use a package like `country-locale-map` that contains all this info:

```suggestion
const clm = require("country-locale-map");

module.exports = clm
  .getAllCountries()
  .flatMap(({
    alpha2: countryCode,
    name: countryName,
    emoji: countryFlag,
    languages,
  }) => languages.map((language) => ({
    label: `${countryFlag} ${countryName} (${language})`,
    value: `${language}_${countryCode}`,
  })));
```

Reply:
There isn't an endpoint for this. Unfortunately, using the country-locale-map includes a lot of languages not supported by Eventbrite and results in errors if they are selected. I added additional labels to the language codes to make them more readable.